cli.js

```diff
@@ -2,14 +2,19 @@
 'use strict';
 var meow = require('meow');
 var supportsColor = require('supports-color');
+var hasFlag = require('has-flag');
 
 meow({
 	help: [
 		'Example',
 		'  $ supports-color',
+		'  $ supports-color --256',
+		'  $ supports-color --16m',
 		'',
 		'Exits with code 0 if color is supported and 1 if not'
 	]
 });
 
-process.exit(supportsColor ? 0 : 1);
+var level = hasFlag('256') ? 'has256' : hasFlag('16m') ? 'has16m' : 'hasBasic';
```

Comment:
No need for `hasFlag` as `meow` parses the CLI arguments: https://github.com/sindresorhus/meow#usage